src/api/print.api.js

```diff
@@ -169,6 +186,8 @@ export class PrintError extends Error {
  *   Default is `false`
  * @param {CoordinateSystem} [config.projection=null] The projection used by the map, necessary when
  *   the grid is to be printed (it can otherwise be null). Default is `null`
+ * @param {String[]} [config.excludedLayers=[]] List of layer names to exclude from the print.
+ *   Default is `[]`
```

Comment:
I would either pass `AbstractLayer` instances, or rename this param `excludedLayerOlNames` to make it clear what it contains

If I understand correctly you will filter layers based on the name we have given them in the OpenLayers context, right?
Might be better to find a way to filter things out based on the layer ID we have in the store (we give each OL layer the same ID as the layer ID in the store, most of the time)

Reply:
I change it to filter by ID, since I guess it's more common to have ID than name. So, I change the param with `excludedLayerIDs` to make it more clear. Is that ok?